src/test/regress/expected/multi_data_types.out

```diff
@@ -285,3 +285,7 @@ SELECT * FROM varchar_hash_partitioned_table WHERE id = 6;
 DROP TABLE composite_type_partitioned_table;
 DROP TABLE bugs;
 DROP TABLE varchar_hash_partitioned_table;
+-- drop all the types created in this test, at once
+SET client_min_messages TO WARNING;
+DROP TYPE test_composite_type, other_composite_type, bug_status CASCADE;
+DROP OPERATOR FAMILY cats_op_fam USING hash;
```

Comment:
I think it's nicer to change this test so it uses a dedicated schema instead? I see you did that for another test. Is that not possible for this one for some reason?

Reply:
Well, this is because dropping a schema that has tables that depend on such composite types was causing bad errors due to https://github.com/citusdata/citus/issues/5780, so I didn't spend much time to create them in a schema.

Reply:
Probably good to add a code comment that we're not using a schema here because of #5780, so future refactoring attempts don't run into the same issue.

Reply:
moving this up in the file and added some explanation on why I did so